src/View/ImageLoader.py

```diff
@@ -39,14 +39,19 @@ def load(self, interrupt_flag, progress_callback):
         # if not ImageLoading.is_dataset_dicom_rt(read_data_dict):
         #    raise ImageLoading.NotRTSetError
 
+        # Initialize the PatientDictContainer singleton.
```

Comment:
This isn't initializing the PatientDictContainer.
It's getting the singleton.  The singleton is responsible for initialising itself.
After this gets the singleton it is (re)populating the singleton, or perhaps re-setting it's contents.